lorahw/gateway.c

```diff
@@ -49,14 +55,16 @@ int set_up_gateway(int com_type, char* path, int region) {
 
     int radio0_freq;
     int radio1_freq;
+    int base_freq;
     switch(region) {
         case 2:
             radio0_freq = EU_RADIO_0_FREQ;
             radio1_freq = EU_RADIO_1_FREQ;
             break;
         default:
-            radio0_freq = US_RADIO_0_FREQ;
-            radio1_freq = US_RADIO_1_FREQ;
+            base_freq = US_BASE_FREQ + (base_channel * 200000);
```

Comment:
[q] Why does the base channel get added to the US base frequency but not the EU frequency?